zerver/webhooks/stripe/doc.md

```diff
@@ -26,7 +26,8 @@ Get Zulip notifications for Stripe events!
 
 !!! tip ""
 
-    Note that Stripe will only accept HTTPS webhooks!
+    Note that Stripe will only accept HTTPS webhooks! Ensure your URL
+    starts with https://.
```

Comment:
So this will only be seen by dev users running the server locally right? I'll suggest this:

> Note that Stripe will only accept HTTPS webhooks! Use tunneling
service such as ngrok or Ultahook if you're running locally.